lms/templates/courseware/courseware-chromeless.html

```diff
@@ -106,3 +106,50 @@
       % endif
     </nav>
 % endif
+
+<script type="text/javascript">
```

Comment:
Why not consolidate this with the earlier iframe check? Appending the `<base>` tag can happen here, right?

Reply:
It can, I just thought repeating myself to keep the functionality isolated would be good. Also it's worth noting that the base tag script needs to be in the `head`. Not a strong reason though. Let me know if you prefer combining them.

Reply:
I put it at the bottom originally so that most of the page would be loaded by the time it ran. It may not be important, but keeping it there may cut down on events as things resize and fill in during load.